app/src/navigation/screens/ClaimScreen.tsx

```diff
@@ -107,16 +108,22 @@ const ClaimScreen: React.FC = () => {
     }
   };
 
-  const isEmail = claim.type === "EmailCredential";
+  const isEmailClaim = claim.type === "EmailCredential";
 
   const onSave = async () => {
     setLoading(true);
-    await addClaim(claim.type, formState, selectedFileIds);
-
-    if (claim.type === "EmailCredential") {
-      const email = (formState.email as string).toLowerCase();
-      await verifyPublicKey(email);
+    if (
+      claim.type === "EmailCredential" &&
+      !isEmail(formState.email as string)
+    ) {
+      Alert.alert(
```

Comment:
Need return here, we dont want to save the email not valid.

Reply:
added a return :)